docs/compute-element/hosted-ce.md

```diff
@@ -44,7 +44,7 @@ Which Communities and Institutions am I Supporting?
 The OSG provides monitoring to view which communities are accessing your site, their fields of science, and home institution.
 Below is an example of the monitoring views that will be available for your cluster.
 
-![monitoring graphs](/compute-element/img/monitoring_graphs.png)
+![monitoring graphs](img/monitoring_graphs.png)
```

Comment:
Why doesn't this one start with a slash, like all the rest?

Reply:
None of them start with a slash any longer since they're all relative links. In this case, the images are kept in an `img/` subri relative to the doc (we're disorganized)